webapp/views/App/SideBar/Modules/ModuleLink/ModuleLink.js

```diff
@@ -28,7 +28,7 @@ const ModuleLink = (props) => {
   const className = getClassName({ root, active })
 
   return (
-    <Link to={uri} className={className} aria-disabled={disabled || active}>
+    <Link to={uri} className={className} aria-disabled={disabled || active} id={`sidebar_${key}`}>
```

Comment:
maybe the id can be a bit more complete, like `sidebar_btn_${key}`

Reply:
Added